### PR TITLE
adding all the mobile docs

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](https://docs.improbable.io/unity/alpha/contributing) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
All the documentation on how to run local deployments for iOS and Android and some additional information.
This is a first version that we publish just to have docs ready and will improve them over time

#### Tests
checked the preview. everything looked good

#### Documentation
this is it

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/content/mobile/ios/local-deploy.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+  1. In the Unity Editor, navigate to **File** > **Build Settings**. Select **Player Settings**, navigate to **Settings for iOS** > **Other Settings** > **Configuration** > **Target SDK** and choose **Device SDK**
+  1.In the Unity Editor, navigate to **File** > **Build Settings** and click **Build**. This prompts you to choose where to save the XCode project that Unity generates. After you've selected the directory, Unity generates the XCode project and opens the folder containing the project. Open the project in XCode, select the Project root, go to **General** > **Signing** and make sure to fix all the errors if there are any.
```
Comment (gkassabli) [inline docs/content/mobile/ios/local-deploy.md]: missing space after 1.

File: docs/content/mobile/ios/setup.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Setting up iOS support for the GDK
+
+## Get the dependencies for developing iOS games
+  1. Follow the steps in [Get the dependencies]({{urlRoot}}/setup-and-installing) and additionally install **iOS build support** for Unity.
+  2. Install iOS prerequisites
```
Comment (gkassabli) [inline docs/content/mobile/ios/setup.md]: Nit: use 1. for consistency.

Comment (jessicafalk) [reply docs/content/mobile/ios/setup.md]: god, I thought I fixed it everywhere

Comment (gkassabli) [reply docs/content/mobile/ios/setup.md]: Sorry, it is my fault for introducing that in the first place :)

File: docs/content/mobile/android/local-deploy.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+  1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
+  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+  > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
```
Comment (DanailPenev) [inline docs/content/mobile/android/local-deploy.md]: Not visualised properly on website.

File: docs/content/mobile/android/local-deploy.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`.  (Where `<your-local-ip>` is the IP address you just located.)
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP and a different launch configuration.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+  1. In the Unity Editor, navigate to **File** > **Build Settings**. Select **Android** as your Build Platform and choose the virtual device from the drop-down menu in the **Run Device** field. The device is likely to be called **Google Android SDK built for x86 (emulator-XXXX)**.
+  1. In the Unity Editor, navigate to **File** > **Build Settings** and click **Build and Run**. (For subsequent runs using the same Emulator, you can just select **File** > **Build and Run**.)
```
Comment (DanailPenev) [inline docs/content/mobile/android/local-deploy.md]: If they navigated in the previous step (which we tell them to), they don't need to navigate again.

Comment (DanailPenev) [reply docs/content/mobile/android/local-deploy.md]: We can put the "For subsequent runs" message in a `>` block below.

File: docs/content/mobile/android/local-deploy.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  1. [Build your server-workers.]({{urlRoot}}/content/build)
+  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP and a different launch configuration.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+  1. In the Unity Editor, go to **File** > **Build Settings**. Choose the type of mobile device from the drop-down menu and click **Build and Run**. (For subsequent runs you can just select **File** > **Build and Run**.)
```
Comment (DanailPenev) [inline docs/content/mobile/android/local-deploy.md]: See comment above.

File: docs/content/mobile/ios/local-deploy.md (hunk 1 above)
Comment (DanailPenev) [inline docs/content/mobile/ios/local-deploy.md]: This is not displayed properly on the website.

File: docs/content/mobile/android/local-deploy.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
+  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+  > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+  1. In the Editor’s Game view, select **Play**.
+    > You can change the resolution of the Game view in your Unity Editor to make sure it does not appear stretched on your mobile device. Choose the resolution that’s identical to your mobile device to produce the best results.
```
[the remark this answers is not in the document]
Comment (jessicafalk) [reply docs/content/mobile/android/local-deploy.md]: there are too many of them 😱

File: docs/content/mobile/android/local-deploy.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP and a different launch configuration.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+  1. In the Unity Editor, go to **File** > **Build Settings**. Choose the type of mobile device from the drop-down menu and click **Build and Run**.
+
+    > For subsequent runs you can just select **File** > **Build and Run**.)
```
Comment (DanailPenev) [inline docs/content/mobile/android/local-deploy.md]: You dropped this here `)`

File: docs/content/mobile/android/local-deploy.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+  1. You need to know the local IP address of your computer to connect. [This page](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) (on the Lifehacker website)  describes how you can find your external and local IP address.
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`.  (Where `<your-local-ip>` is the IP address you just located.)
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP and a different launch configuration.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+  1. In the Unity Editor, navigate to **File** > **Build Settings**. Select **Android** as your Build Platform and choose the virtual device from the drop-down menu in the **Run Device** field. The device is likely to be called **Google Android SDK built for x86 (emulator-XXXX)**.
+  1. In the Unity Editor, navigate to **File** > **Build Settings** and click **Build and Run**.
```
Comment (DanailPenev) [inline docs/content/mobile/android/local-deploy.md]: Still thinking we shouldn't say the same step twice in a row.

File: docs/content/mobile/ios/local-deploy.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+## Connecting your iOS simulator to a local deployment
+
+  1. [Build your server-workers.]({{urlRoot}}/content/build)
+  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json`.
+
+    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify a different launch configuration.
+    >
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
+    1. In the Unity Editor, navigate to **Edit** > **Project Settings** > **Player** > **Settings for iOS** > **Other Settings** > **Configuration** > **Target SDK** and choose **Simulator SDK**.
```
Comment (DanailPenev) [inline docs/content/mobile/ios/local-deploy.md]: (LINE 37) this is broken on the website.

File: docs/content/mobile/android/ways-to-test.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+  * [Setting up iOS support for the GDK]({{urlRoot}}/content/mobile/android/setup)
+
+Unity provides multiple ways to test your Android [client-worker]({{urlRoot}}/content/glossary#client-worker). We integrated them all to work with [SpatialOS]({{urlRoot}}/content/glossary#spatialos-runtime). This documentation describes the benefits of the different options.
+
+## In the Editor
+For standard workflows and for minor changes, run your game in the Editor. When the build platform is set for Android, it executes code that is in preprocessor `#if UNITY_ANDROID`clauses. This way, you have the full capabilities and ease of use of the Unity Editor, while still testing code that would otherwise only run on a mobile device.
+
+## Unity Remote
+
+With the Unity Remote, you don’t have to spend time building and deploying your game, reducing development iteration times. It mirrors the Unity Editor’s Game view on your mobile device, giving you quick feedback on how the game looks on a mobile device. However, it does not provide the full native capabilities the game running on a device.
```
Comment (paulbalaji) [inline docs/content/mobile/android/ways-to-test.md]: full native capabilities **of** the game running on a device

Comment (paulbalaji) [inline docs/content/mobile/android/ways-to-test.md]: full native capabilities **of** the game running on a device

File: docs/content/mobile/ios/ways-to-test.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+  * [Setting up iOS support for the GDK]({{urlRoot}}/content/mobile/ios/setup)
+
+Unity provides multiple ways to test your iOS [client-worker]({{urlRoot}}/content/glossary#client-worker). We integrated them all to work with [SpatialOS]({{urlRoot}}/content/glossary#spatialos-runtime). This documentation describes the benefits of the different options.
+
+## In the Editor
+For standard workflows and for minor changes, run your game in the Editor. When the build platform is set for iOS, it executes code that is in preprocessor `#if UNITY_IOS` clauses. This way, you have the full capabilities and ease of use of the Unity Editor, while still testing code that would otherwise only run on a mobile device.
+
+## Unity Remote
+
+With the Unity Remote, you don’t have to spend time building and deploying your game, reducing development iteration times. It mirrors the Unity Editor’s Game view on your mobile device, giving you quick feedback on how the game looks on a mobile device. However, it does not provide the full native capabilities the game running on a device.
```
Comment (paulbalaji) [inline docs/content/mobile/ios/ways-to-test.md]: full native capabilities **of** the game running on a device

File: docs/content/modules/core-and-feature-module-overview.md [hunk 12]
```diff
@@ -89,3 +89,22 @@ Use this enable custom spawning by passing in parameters to change the default.
...
+To access this module, use the `Improbable.Gdk.Mobile` namespace. It offers support to connect mobile [client-workers]({{urlRoot}}/content/glossary#client-worker) to SpatialOS.
+
+`Improbable.Gdk.Mobile` is in the repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.mobile).
+
+This module consists of:
+
+* `MobileWorkerConnector` - in the repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.mobile).<br/>
+Inherit from this class to define your custom mobile worker connectors.
+
+*    `Improbable.Gdk.Mobile.Android` - in the repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.mobile/Android).
```
Comment (paulbalaji) [inline docs/content/modules/core-and-feature-module-overview.md]: spacing after *

**Pull request merged**